### ISSUE TITLE
Attempted to divide by zero when calling SentrySdk.Init

### ISSUE THREAD

**Issue by acosta32** [getsentry/sentry-dotnet#954]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io) 

Which SDK and version?
.Net 3.3.2

### Steps to Reproduce

Just call SentrySdk.Init("url")

### Expected Result

No error

### Actual Result

Exception message: Attempted to divide by zero.
Stack trace: at Sentry.Internal.ProcessInfo..ctor(SentryOptions options, Func1 findPreciseStartupTime) at Sentry.SentrySdk.InitHub(SentryOptions options) at Sentry.SentrySdk.Init(Action1 configureOptions)

**Comment by bruno-garcia** [comment]
@acosta32 I can't reproduce this. Could you please share a complete repro?

I'm running:

```csharp
using (SentrySdk.Init("https://80aed643f81249d4bed3e30687b310ab@o447951.ingest.sentry.io/5428537"))
{
    // The following exception is captured and sent to Sentry
    throw null;
}
```

And it works fine

**Comment by bruno-garcia** [comment]
That must be coming from here:

```
            BootTime = now.AddTicks(-Stopwatch.GetTimestamp()
                                    / (Stopwatch.Frequency
                                       / TimeSpan.TicksPerSecond));
```

What operating system and version of .NET are you using?

**Comment by bruno-garcia** [comment]
One value is `const` and the other a `static readonly` part of the .NET class library. Not sure how this is ending up as a device by 0

**Comment by acosta32** [comment]
It happens only on one of the servers. Windows 2012

**Comment by acosta32** [comment]
More info:

Stopwatch.Frequency = 2048408
TimeSpan.TicksPerSecond  = 10000000
(Stopwatch.Frequency/ TimeSpan.TicksPerSecond)  = 0

**Comment by bruno-garcia** [comment]
Opened a PR to address this ^.

You'd expect that running tests on Mono, .NET Core and .NET Framework, on Windows, macOS and Linux would be enough to catch these types of errors.

**Comment by bruno-garcia** [comment]
Thanks for the details. Seems like something is broken there. Check out the .NET source:

https://github.com/dotnet/runtime/blob/4a2a44a6179576ce7d033cd43626843b39a84ec9/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Stopwatch.cs#L12-L13

**Comment by acosta32** [comment]
Problem seems to be Stopwatch.Frequency which calls

public static readonly long Frequency = QueryPerformanceFrequency();

Interop.BOOL result = Interop.Kernel32.QueryPerformanceFrequency(&resolution);

Value is too low and division goes < 1 which then is cast to long as 0

**Comment by bruno-garcia** [comment]
Can you please check: `Stopwatch.IsHighResolution`? what does it return?

**Comment by acosta32** [comment]
Returns True on both good and bad Servers

**Comment by bruno-garcia** [comment]
Released as 3.3.3: https://github.com/getsentry/sentry-dotnet/releases/tag/3.3.3

Thanks for reporting
